packages/cli/src/runners/task-managers/data-request-response-stripper.ts

```diff
@@ -11,6 +11,7 @@ import type {
  * Strips data from data request response based on the specified parameters
  */
 export class DataRequestResponseStripper {
+	// @Reviewer: Same question here, do we need this for node types? Maybe not since they're more "default" than node names?
```

Comment:
AFAIK no changes needed here. You could confirm that this works by running `$prevNode.nodeType` in a Code Node and checking that it works

Reply:
Confirmed it works, thank you!